test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp

```diff
@@ -35,7 +35,8 @@ static auto dna4_01 = []()
         // ---------ACGTACG-TA
```

Comment:
Add (this should also when be available in semi_global_edit_distance_max_errors_unbanded.hpp)

```patch
diff --git a/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp b/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp
index 1a2f51221..12798dc6c 100644
--- a/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp
+++ b/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp
@@ -24,6 +24,10 @@ namespace seqan3::test::alignment::fixture::semi_global::edit_distance::unbanded
 using seqan3::detail::column_index_type;
 using seqan3::detail::row_index_type;
 
+static constexpr auto semi_global_edit_distance = seqan3::align_cfg::method_global
+                                                | seqan3::align_cfg::edit
+                                                | seqan3::align_cfg::aligned_ends{seqan3::free_ends_first};
+
 static auto dna4_01 = []()
 {
     return alignment_fixture
```

You may want to add this line, to make it clear where we imported that from.

```patch
diff --git a/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp b/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp
index 1a2f51221..693452b0a 100644
--- a/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp
+++ b/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_unbanded.hpp
@@ -24,6 +24,8 @@ namespace seqan3::test::alignment::fixture::semi_global::edit_distance::unbanded
 using seqan3::detail::column_index_type;
 using seqan3::detail::row_index_type;
 
+static constexpr auto semi_global_edit_distance = seqan3::align_cfg::method_global;
+
 static auto dna4_01 = []()
 {
     return alignment_fixture

```

Reply:
I applied the first patch but I don't what you mean by the second patch?

Reply:
Sorry my second patch was faulty, I meant 

```patch
diff --git a/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_max_errors_unbanded.hpp b/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_max_errors_unbanded.hpp
index 3f7af4d44..5d7c6737e 100644
--- a/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_max_errors_unbanded.hpp
+++ b/test/unit/alignment/pairwise/fixture/semi_global_edit_distance_max_errors_unbanded.hpp
@@ -26,6 +26,7 @@ namespace seqan3::test::alignment::fixture::semi_global::edit_distance::max_erro
 using namespace seqan3::test::alignment::fixture::semi_global::edit_distance::unbanded;
 using seqan3::detail::column_index_type;
 using seqan3::detail::row_index_type;
+using seqan3::test::alignment::fixture::semi_global::edit_distance::unbanded::semi_global_edit_distance;
 
 static auto dna4_01_e255 = []()
 {
```